brian2/parsing/expressions.py

```diff
@@ -336,6 +338,11 @@ def parse_expression_dimensions(expr, variables):
             u = left_dim*right_dim
         elif op == 'Div':
             u = left_dim/right_dim
+        elif op == 'FloorDiv':
+            if not (left_dim is DIMENSIONLESS and right_dim is DIMENSIONLESS):
+                raise SyntaxError('Floor division can only be used on '
+                                  'dimensionless integer values.')
```

Comment:
Can also be used on non-integer values? In principle if the output is dimensionless it still makes sense?

Reply:
Yes, indeed.